galasa-ui/src/components/TokenResponseModal.tsx

```diff
@@ -39,13 +62,14 @@ export default function TokenResponseModal({ refreshToken, clientId, onLoad }: T
       preventCloseOnClickOutside
       onRequestClose={() => {
         setOpen(false);
+        deleteClientIdAndTokenOnLoad()
```

Comment:
This isn't happening when the component loads anymore. If the user doesn't close the modal, the cookies won't be deleted.